### PR TITLE
OCCT.OCCT version 11.0.11

### PR DESCRIPTION
**Pull request by thejump24**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/71811)

### CODE REVIEW

File: manifests/o/OCCT/OCCT/11.0.11/OCCT.OCCT.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: OCCT.OCCT
+PackageVersion: 11.0.11
+Installers:
+- Architecture: x64
+  InstallerType: exe
```
Comment (Trenly) [inline manifests/o/OCCT/OCCT/11.0.11/OCCT.OCCT.installer.yaml:8]:
```suggestion
  InstallerType: portable
```

File: manifests/o/OCCT/OCCT/11.0.11/OCCT.OCCT.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: OCCT.OCCT
+PackageVersion: 11.0.11
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://www.ocbase.com/download/edition:Personal
+  InstallerSha256: 9718DF71C813B56515284FE60A620F979934004BA10FD2A2448CD9AF622A6E5C
+  InstallerSwitches:
```
Comment (Trenly) [inline manifests/o/OCCT/OCCT/11.0.11/OCCT.OCCT.installer.yaml:11]: Installer switches not necessary for portable apps

File: manifests/o/OCCT/OCCT/11.0.11/OCCT.OCCT.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: OCCT.OCCT
+PackageVersion: 11.0.11
+PackageLocale: en-US
+Publisher: OCCT - Ocbase - Adrien Mercier
+PackageName: OCCT
+License: OCCT license
```
Comment (Trenly) [inline manifests/o/OCCT/OCCT/11.0.11/OCCT.OCCT.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by Trenly — CHANGES_REQUESTED** [review]
InstallationVerification-x64-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20220824015358

I believe this is a portable application.

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Inactive PR;

**Pull request closed without merging**